### ISSUE TITLE
Make DropBox integration optional

### ISSUE THREAD

**Issue by techryda** [todotxt/todo.txt-android#354]
Firstly, thank you for writing such a great app/standard!

My issue: The Android app is currently unusable by me as I do not have/wish to have a DropBox account.  Can you make the DropBox integration/syncing features optional? I use OwnCloud and like to retain control over my own data.  I'm not asking that you support OwnCloud (unless you plan to) but just to give the option to privacy conscious people to use your software.

Basically I'd like to see ToDo.txt Touch just read and write to a todo.txt file on my device.  I'll manage syncing/backups.

Thanks
